src/dct/systems/IADS.lua

```diff
@@ -175,12 +167,10 @@ function IADS:enableSAM(site)
 			return
 		end
 	end
-
```

Comment:
Do not see why we need to remove empty lines like this, reason?

Reply:
No reason, looks nicer.